Common/src/main/java/vazkii/botania/common/block/mana/BlockPrism.java

```diff
@@ -120,6 +120,8 @@ public InteractionResult use(BlockState state, Level world, BlockPos pos, Player
 		} else if (!lens.isEmpty()) {
 			player.getInventory().placeItemBackInInventory(lens);
 			prism.getItemHandler().setItem(0, ItemStack.EMPTY);
+		} else {
+			return InteractionResult.PASS;
 		}
 
 		return InteractionResult.SUCCESS;
```

Comment:
I'm surprised java or intellij did not complain about dead code on this line

Reply:
It's not dead, the if above the addition doesnt return